src/Dev/SapphireTest.php

```diff
@@ -283,9 +283,6 @@ protected function setUp()
         $fixtureFiles = $this->getFixturePaths();
 
         if ($this->shouldSetupDatabaseForCurrentTest($fixtureFiles)) {
-            /** @var FixtureTestState $fixtureState */
-            $fixtureState = static::$state->getStateByName('fixtures');
-            $this->setFixtureFactory($fixtureState->getFixtureFactory(static::class));
```

Comment:
We still need this line because some old tests use the prop not just the accessor.

Reply:
Can I make that more explicit by assigning `$this->fixtureFactory = $fixtureState->getFixtur...` rather than setting the getting?

Reply:
Well, you could if you want, but that's what the setter does right.

Reply:
Yeah it does, but it also sets the fixture factory to the FixtureTestState which is why I got confused by this initially

Reply:
Updated with new commit =)